src/components/Company/index.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Company extends Component {
+
+    render() { 
+        const {company} = this.props.company;
```

Comment:
const{company} = this.props;

Reply:
@minamrp fixed the issue, see details:
From Experience component we receive "experience" as props:
```
{experience.map((item, index) => {
                        return (
                            <Company key={index} experience={item} />
                        );
                    })}
```

Passing it to Company as such:
`const {company} = this.props.experience;`

Thanks for your help!